client/src/index.html

```diff
@@ -3,15 +3,23 @@
 	<head>
 		<meta charset="UTF-8" />
 		<meta name="viewport" content="initial-scale=1, width=device-width" />
-		<title>Starter Kit</title>
+		<title>CYF Jobs Board</title>
 	</head>
 	<body>
 		<noscript>You need to enable JavaScript to run this app.</noscript>
-		<div id="root"></div>
-		<script
-			crossorigin
-			type="text/javascript"
-			src="https://polyfill.io/v3/polyfill.min.js?flags=gated&features=es6"
-		></script>
+		<header>
+			<!-- Logo and navigation links go here -->
+		</header>
+		<main>
+			<div id="root"></div>
+			<script
+				crossorigin
+				type="text/javascript"
+				src="https://polyfill.io/v3/polyfill.min.js?flags=gated&features=es6"
+			></script>
+		</main>
+		<footer>
+			<!-- Footer content goes here -->
+		</footer>
```

Comment:
@Meysam-Arshadi, I think Oliver was referring to the layout component  not the root.

Reply:
Thanks @OlushoreOdude. Done!